manifests/config.pp

```diff
@@ -7,19 +7,21 @@
   $runners    = $gitlab_ci_multi_runner::runners,
 ) {
 
-  concat { $::gitlab_ci_multi_runner::config_file:
+  concat { $gitlab_ci_multi_runner::config_file:
     ensure  => present,
-    owner   => $::gitlab_ci_multi_runner::user,
-    group   => $::gitlab_ci_multi_runner::group,
+    owner   => $gitlab_ci_multi_runner::user,
+    group   => $gitlab_ci_multi_runner::group,
     require => Package[$gitlab_ci_multi_runner::package_name],
     notify  => Service[$gitlab_ci_multi_runner::service_name],
   }
 
-  concat::fragment { "header_$::gitlab_ci_multi_runner::config_file":
-    target  => $::gitlab_ci_multi_runner::config_file,
+  concat::fragment { "header_${::gitlab_ci_multi_runner::config_file}":
+    target  => '/etc/gitlab-runner/config.toml',
```

Comment:
My bad, I must have changed that while I was testing the whole module. I've send another push with the correct variable path.